misc/plot-roh.py

```diff
@@ -352,6 +359,16 @@ def parse_samples(fname,highlight):
 off = 0
 off_sep = 0
 dat_rg1 = {}
+
+if not chrs:
+    print("No GT tags found in input file.\n\nPlease try rerunning using run-roh.pl with --roh-args \"-G 30\"")
```

Comment:
This is misleading, the -G30 option is for something else. It's for when the run-roh.pl script fails because there are no FORMAT/PL tags in the VCF.

When there are no GT lines present in the input to plot-roh.py, that would be caused by the user trying to feed in raw output of `bcftools roh`. So the suggestion should be something like "there are no GT lines in the input file, was it generated using the run-roh.pl script?"

Reply:
Okay, I misunderstood.  I will change it.

Reply:
Done ...